app/src/main/java/org/gateshipone/malp/application/fragments/EditProfileFragment.java

```diff
@@ -73,11 +75,33 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mProfilenameView = (TextInputEditText) rootView.findViewById(R.id.fragment_profile_profilename);
         mHostnameView = (TextInputEditText) rootView.findViewById(R.id.fragment_profile_hostname);
         mPasswordView = (TextInputEditText) rootView.findViewById(R.id.fragment_profile_password);
-        mPortView = (NumberPicker) rootView.findViewById(R.id.fragment_profile_port);
+        mPortView = (TextInputEditText) rootView.findViewById(R.id.fragment_profile_port);
```

Comment:
mPortView is of wrong type (NumberPicker) here. Needs to be TextInputEditText also